integration/main_test.go

```diff
@@ -42,7 +42,26 @@ import (
 	"github.com/awslabs/soci-snapshotter/util/testutil"
 )
 
-const enableTestEnv = "ENABLE_INTEGRATION_TEST"
+const (
+	enableTestEnv         = "ENABLE_INTEGRATION_TEST"
+	containerdLogLevelEnv = "CONTAINERD_LOG_LEVEL"
+	sociLogLevelEnv       = "SOCI_LOG_LEVEL"
+)
+
+// this can be overwritten by setting up env variables specified by
+// `containerdLogLevelEnv`/`sociLogLevelEnv`.
+var (
+	containerdLogLevel = "warn"
+	sociLogLevel       = "debug"
+)
+
+func validateLogLevel(level string) bool {
+	switch level {
+	case "trace", "debug", "info", "warn", "error", "fatal", "panic":
```

Comment:
nit: What do you think about replacing this with a logrus check? 
```
_, err := logrus.ParseLevel(level);
return err == nil;
```

Reply:
sure, not aware of this :) got the list of value from cli help output 😅